demo/admin/src/products/future/ProductForm.cometGen.ts

```diff
@@ -38,6 +38,8 @@ export const ProductForm: FormConfig<GQLProduct> = {
                 { type: "boolean", name: "inStock" },
                 { type: "date", name: "availableSince" },
                 { type: "block", name: "image", label: "Image", block: { name: "DamImageBlock", import: "@comet/cms-admin" } },
+                { type: "file", name: "priceList", label: "Price List", maxFileSize: 1024 * 1024 * 4 },
```

Comment:
Will we also get a "damFile" type? (that opens a Dam dialog to select a file)

Is the naming ("file" and "damFile") still good then?

Reply:
> Will we also get a "damFile" type? 

There is nothing planned, that I know of. 
@Ben-Ho maybe?

> Is the naming ("file" and "damFile") still good then?

`file` to me is very generic, which seems to be correct for this.

Reply:
Maybe fileUpload is better, which is more related to fileUploads?

Reply:
Renamed in ecf820de6886e296be5c13b1daa11bc02fbed3a2